### DISCUSSION TITLE
Mono: Core API hash mismatch

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [godotengine/godot-proposals#4709] [Scripting]
There are so many issues involving Mono: Core API hash mismatch

Can someone please explain why this often happen and where and how to troubleshoot?

[Mono: Core API hash mismatch](https://github.com/godotengine/godot/issues/52250)

[Mono always gives Core API hash mismatch even with glue from same commit](https://github.com/godotengine/godot/issues/14584)

[[Mono] Windows export template - Core API hash mismatch](https://github.com/godotengine/godot/issues/31581)

[Error with Exported C# Game: "Mono: Core API hash mismatch](https://github.com/godotengine/godot/issues/35290)

**Comment by Calinou** [comment]
Please don't ask support questions here, as this issue tracker is meant to be used for feature proposals only. Use one of the other [community channels](https://godotengine.org/community) instead.
